windows/CodePushPackage.cs

```diff
@@ -0,0 +1,364 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.IO;
+using System.IO.Compression;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Storage;
+using Windows.Storage.Streams;
+using Windows.Web.Http;
+
+namespace CodePush.ReactNative
+{
+    class CodePushPackage
```

Comment:
Yea this file isn't really a package, it's a collection of methods that deal with packages/updates. I like "UpdateManager"